global/ci-infra.repos

```diff
@@ -0,0 +1,195 @@
+%YAML 1.1
```

Comment:
Do we want to host the repos file in the ros_buildfarm_config repository? I wouldn't want to create a repo just for it but I think https://github.com/ros-infrastructure/ci would also be a candidate or hosting repos files as part of the repository and web site that we'll need to host for the OSRA Infrastructure charter compliance.

I'm mostly concerned with scale if we begin to add other global CI to the build farm but that concern is likely unfounded so this is not a binding question.

Reply:
That sounds fine to me. It also decouples this change from ros-infrastructure/ros_buildfarm#1057.